websites/A/Animeler.net/dist/metadata.json

```
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Chas",
+		"id": "498195150711947285"
+	},
+	"service": "Animeler.net",
+	"description": {
+		"en": "Animeler.net is a website that you can reach latest news about animes. Provided by Turk Anime TV.",
+		"tr": "Animeler.net, anime haberlerini size en hızlı şekilde sunan sitedir. Türk Anime TV iştirakidir.",
```

This is the official description in Turkish

```suggestion
		"tr": "En iyi, en güncel, en çok konuşulan içerikler ve farklı kaynaklardan tamamen tarafsız içerikleri okuyun. Türk Anime TV iştirakidir.",
```